src/SFA.DAS.ApplyService.Web/Services/NotRequiredOverridesService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.Extensions.Options;
+using Newtonsoft.Json.Linq;
+using SFA.DAS.ApplyService.Session;
+using SFA.DAS.ApplyService.Web.Configuration;
+using SFA.DAS.ApplyService.Web.Infrastructure;
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.ApplyService.Web.Services
+{
+    public class NotRequiredOverridesService : INotRequiredOverridesService
```

Comment:
Can't see any specific tests for this. Are they present?

Reply:
Added tests for this service